projects/plugins/super-cache/wp-cache-phase2.php

```diff
@@ -509,13 +509,13 @@ function wpsc_get_accept_header() {
 	static $accept = 'N/A';
 
 	if ( $accept === 'N/A' ) {
-		$json_list = array( 'application/json', 'application/activity+json', 'application/ld+json' );
+		$json_list = apply_filters( 'wpsc_accept_list', array( 'application/json', 'application/activity+json', 'application/ld+json' ) );
```

Comment:
Based on the filter name - `$json_list` is `accept list`. Or `$accepted_headers`.

Reply:
Thanks. Did that, and renamed the filter too to "wpsc_accept_headers" to reflect what the filter is actually working on.